### PR TITLE
Update Get-WebApplicationProxyConfiguration

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR replaces the example output which is from WAP 2012 R2 with example output produced from WAP 2016.

### CODE REVIEW

**Comment by claydetels19** [comment]
Hi @e0i   - We understand that these [PR criteria](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master) are new, and we appreciate your patience with the process. 

**Following are blocking issues that need to be resolved before we can merge:**

Get-WebApplicationProxyConfiguration.md
- The content preview must be clean on staging. It cannot contain any obvious formatting issues.



***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**
- The content preview must be clean on staging. It cannot contain any obvious formatting issues.  Note:  first line of characters appears to run off the screen.




***

If this was intentional please resubmit.

Thanks,

Clay

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

File: docset/windows/webapplicationproxy/Get-WebApplicationProxyConfiguration.md [hunk 1]
```diff
@@ -41,61 +41,33 @@ To modify the settings, use the Set-WebApplicationProxyConfiguration cmdlet.
 PS C:\>Get-WebApplicationProxyConfiguration
 
 ADFSTokenSigningCertificatePublicKey   : MIIFPzCCBCegAwIBAgIKJ95AmAAAAAA0djANBgkqhkiG9w0BAQUFADAWMRQwEgYDVQQDEwtBRFNUU1
```
Comment (JohanFreelancer9) [inline docset/windows/webapplicationproxy/Get-WebApplicationProxyConfiguration.md]:
```suggestion
ADFSTokenSigningCertificatePublicKey: MIIFPzCCBCegAwIBAgIKJ95AmAAAAAA0djANBgkqhkiG9w0BAQUFADAWMRQwEgYDVQQDEwtBRFNUU1
```

**Comment by X-Guardian** [comment]
All I've done is removed the blank lines from the output. I've not changed the formatting of the original output at all.

Looking deeper at this output, it is from 2012 R2 anyway, 2016 outputs more properties. See [AppProxyGlobalConfiguration](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/appproxypsprov/appproxyglobalconfiguration). I'll change this PR to have a correct 2016 output example.

**Comment by X-Guardian** [comment]
I can't see why the build is failing and I have no access to https://review.docs.microsoft.com to troubleshoot.

**Comment by traya1** [comment]
Closing/Re-opening the PR

**Comment by traya1** [comment]
Hi @X-Guardian, the validation check has passed. Please remove the "Back to Submitter" label and add "SignOff" to have the PR reviewed. Thanks!

**Comment by X-Guardian** [comment]
Hi @traya1, I don't have permissions to update labels on this PR.

**Comment by traya1** [comment]
Oh ok, I have updated my comment to include the correct person with permissions.

**Comment by e0i** [comment]
@X-Guardian Thank you for your contributions and following up with the revision suggestions.

**Pull request merged**